src/utils/queuedTxsActions.ts

```diff
@@ -0,0 +1,43 @@
+import { isSignableBy, isTransactionListItem } from '@/utils/transaction-guards'
+import type { TransactionListPage } from '@safe-global/safe-gateway-typescript-sdk'
+
+export type SafeTxsActions = {
+  queued?: string | number
+  signing?: string | number
```

Comment:
Why are they a string or a number? Just stick with a string if you need to say `10+`.

Naming-wise, `totalQueued` or `queuedCount` would be better.

`signing` doesn't work at all. It's the number of txs you need to sign. They are not in the process of being signed by you. I would call it `totalToSign`.